src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -777,6 +782,47 @@ public void canSearchForItemsByBarcode()
       is("673274826203"));
   }
 
+  @Test
+  public void canSearchForItemsByTags()
```

Comment:
Personally, I would also update the tests for creating and updating items to include tags as well.

(I think that is also relevant for the previous changes for instances, apologies that I didn't manage to provide that feedback during those pull requests)

Reply:
Thanks it really makes sense.